api/v1alpha3/zz_generated.deepcopy.go

```diff
@@ -1,4 +1,3 @@
-//go:build !ignore_autogenerated
```

Comment:
looks like our controller-gen in ci still adds these :-)

Reply:
I don't know what happened .. need further analysis
it should not report such error as I thought it's fixed already